code/_onclick/telekinesis.dm

```diff
@@ -32,18 +32,24 @@ var/const/tk_maxrange = 15
 		return
 
 	var/obj/item/tk_grab/O = new(src)
-	user.put_in_active_hand(O)
-	O.host = user
-	O.focus_object(src)
+	if(user.put_in_hands(O))
+		user.put_in_active_hand(O)
```

Comment:
idk why it works.

The behavior currently acts like it picks up the item, and it glows like you're moving it with TK, but it's immediately dropped before the icon even shows up in hand on the player's end.

As-is the code works where it doesn't show you the shinies and gives you the message, you're free to help me find out why.